src/Lasso.jl

```diff
@@ -301,6 +301,10 @@ function StatsBase.fit{T<:AbstractFloat,V<:FPVector}(::Type{LassoPath},
     # Standardize predictors if requested
     if standardize
         Xnorm = vec(full(std(X, 1, corrected=false)))
+        if findfirst(Xnorm .== zero(T)) != 0 
+            warn("One of the predicators (columns of X) is a constant, so it can not be standardized. 
+                  To include a constant predicator set standardize = false and intercept = false") 
```

Comment:
To avoid printing the second line with an indent, `"""` quotes might be better than `"`.